src/client/routes/events/ManageEventTypes.ts

```diff
@@ -0,0 +1,19 @@
+import { MutationFunction } from '@apollo/react-common';
+import {
+	AddOrUpdateEventMutation,
+	AddOrUpdateEventMutationVariables,
+} from '../../generated/graphql';
+
+export interface EventUpdate {
+	name: string;
+	startTimestamp: string;
```

Comment:
what's the format of a sample timestamp? is it like "1519129853500" or "January 1, 1970, 00:00:00 UTC"

Reply:
Added example